vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -431,24 +436,30 @@ form {
 }
 
 @media only screen and (min-width: $lapWidth) {
+  .form_item_horiz legend,
   .form_item_horiz_label,
   .form_item_horiz_input {
     display: inline-block;
     vertical-align: top;
   }
 
+  .form_item_horiz legend {
+    float: left;
+  }
+
+  .form_item_horiz legend,
   .form_item_horiz_label {
     text-align: right;
-    width: 25%;
-    padding-right: $rhythm * 4;
+    width: 30%;
+    padding-right: $lineHeight;
   }
 
   .form_item_horiz_input {
-    width: 75%;
+    width: 70%;
   }
 
   .form_actions.horizontal {
-    padding-left: 24.99999%;
+    padding-left: 30%;
```

Comment:
I understand. But why the change from 25% to 30%? Are we running out of room somewhere? Just trying to understand how this fits into the fieldset stuff.